packages/tokens/terrazzo.config.js

```diff
@@ -1,11 +1,13 @@
 import { defineConfig } from '@terrazzo/cli';
 import { makeCSSVar } from '@terrazzo/token-tools/css';
 import css from '@terrazzo/plugin-css';
+import sass from '@terrazzo/plugin-sass';
```

Comment:
Note: this is fine for locally-testing, but all tests should be in `plugin-sass` and this package shouldn’t be modified at all. But feel free to copy examples from this into the `plugin-sass` tests just for testing